WhateverGreen/kern_igfx.cpp

```diff
@@ -820,9 +830,10 @@ bool IGFX::applyPlatformInformationListPatch(uint32_t framebufferId, T *platform
 		DBGLOG("igfx", "stolenMemorySize: 0x%08X", frame->fStolenMemorySize);
 		DBGLOG("igfx", "framebufferMemorySize: 0x%08X", frame->fFramebufferMemorySize);
 		DBGLOG("igfx", "unifiedMemorySize: 0x%08X", frame->fUnifiedMemorySize);
-		r = true;
```

Comment:
Ugh, I think this was deleted by accident.

Reply:
Or rather, not added back...